misk/build.gradle

```diff
@@ -9,6 +9,7 @@ kotlin {
 }
 apply plugin: 'org.junit.platform.gradle.plugin'
 apply plugin: 'com.vanniktech.maven.publish'
+apply from: "https://raw.githubusercontent.com/adrw/misk/4e5ce9df6d3057efcee9dcd185623c69bfe39599/gradle/web.gradle"
```

Comment:
Can we reference the master branch of the `square` user instead?

Reply:
Yeah I was going to put that in a follow up PR but I don't have a good hash in Misk yet to reference.

Reply:
I would lean to doing two prs for this then